Hermes/appscale/hermes/stats/producers/proxy_stats.py

```diff
@@ -229,6 +228,8 @@ class ProxyStats(object):
   backend = attr.ib(metadata={Meta.ENTITY: HAProxyBackendStats})
   servers = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyServerStats})
   listeners = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyListenerStats})
+  servers_count = attr.ib()  # number of servers serving this proxy
+  listeners_count = attr.ib()  # number of listeners serving this proxy
```

Comment:
You're right, but alternative way to get these values is to count length of servers and listeners lists. So alternative way requires client to request list of servers or list of listeners. During my autoscale testing and Maxim's development of stats on tools side we realized that it's useful to know number of servers without digging into servers details.